tests/Carbon/TestingAidsTest.php

```diff
@@ -111,18 +111,39 @@ public function testParseRelativeWithTestValueSet()
 
             $scope->assertSame('2013-10-01 05:15:05', Carbon::parse('first day of next month')->toDateTimeString());
             $scope->assertSame('2013-09-30 05:15:05', Carbon::parse('last day of this month')->toDateTimeString());
+
+            $scope->assertSame('2013-09-01 06:00:00', Carbon::parse('6:00')->toDateTimeString());
+            $scope->assertSame('2013-09-01 17:31:24', Carbon::parse('17:31:24')->toDateTimeString());
+
+            $scope->assertSame('2013-09-01 12:00:00', Carbon::parse('noon')->toDateTimeString());
         }, $testNow);
     }
 
-    public function testHasRelativeKeywords()
+    public function testParseRelativeWithTestValueSetAndMicroseconds()
     {
-        $this->assertFalse(Carbon::hasRelativeKeywords('sunday 2015-02-23'));
-        $this->assertTrue(Carbon::hasRelativeKeywords('today +2014 days'));
-        $this->assertTrue(Carbon::hasRelativeKeywords('next sunday -3600 seconds'));
-        $this->assertTrue(Carbon::hasRelativeKeywords('last day of this month'));
-        $this->assertFalse(Carbon::hasRelativeKeywords('last day of last month of 2015'));
-        $this->assertTrue(Carbon::hasRelativeKeywords('first sunday of next month'));
-        $this->assertFalse(Carbon::hasRelativeKeywords('first sunday of January 2017'));
```

Comment:
No test removing until 2.x

Reply:
These tests were moved to RelativeDateStringTest. Only `last day of last month of 2015` was removed because it's invalid (`new DateTime('last day of last month of 2015')` throws exception).